tests/linearGaussian_snre_test.py

```diff
@@ -65,8 +66,9 @@ def test_api_nre_multiple_trials_and_rounds_map(
         for num_trials in [1, 3]:
             x_o = zeros((num_trials, num_dim))
             posterior = inference.build_posterior(
-                mcmc_method="slice_np_vectorized",
-                mcmc_parameters=mcmc_params_fast,
+                posterior_parameters=MCMCPosteriorParameters(
+                    method="slice_np_vectorized", **mcmc_params_fast
+                )
```

Comment:
we could also think about changing the fixture `mcmc_params_fast` from returning a dict to returning the `MCMCPosteriorParamters` instance with `method="slice_np_vectorized"`? 
But note that there are some edge cases where we explicitly set `method="slice_np"`, but only 1-2 I believe.